index.html

```diff
@@ -48,30 +51,182 @@ <h2>I'm a Software Developer</h2>
+      </section>
+      <section class="projthirdpage">
+        <div id="project5">
+          <h2 class="work-subtitle protitle">The Quiz</h2>
+          <p class="work-details prodetails">Make a quiz templete using html forms</p>
+          <div class="flex">
+           <p class="flex-item item1 proitem1">css</p>
+           <p class="flex-item item2 proitem2">html</p>
+           <p class="flex-item item3 proitem3">bootstrap</p>
+          </div>
+        </div>
+        <button class="work-button probutton">See Project</button>
+        <div id="project6">
+          <h2 class="work-subtitle protitle pro2title">Tribute Page</h2>
+          <p class="work-details prodetails">Tribute to Dr. Abdul Salam a Nobel Scientist.</p>
+          <div class="flex pro2flex">
+           <p class="flex-item item1 proitem1">css</p>
+           <p class="flex-item item2 proitem2">html</p>
+           <p class="flex-item item3 proitem3">bootstrap</p>
+          </div>
+        </div>
+        <button class="work-button probutton">See Project</button>
+      </section>
+      <section id="aboutme">
+        <h1 id="me">About me</h1>
+        <p>
+          I can help you to build a product, feature or website Look through
+          some of my work and experience! If you like what you see and have a
+          project you need coded, don't hesitate to contact me.
+        </p>
+        <button class="work-button resumebtn">Get My Resume</button>
+        <div><img id="aboutmeimg" src="/img/illustration-about-me-mobile.png" alt=""></div>
+        </section>
+        <section id="languages">
+          <div class="thumbnail">
+          <div class="rectangle">
+          </div>
+          <h1 class="headings">Languages</h1>
+        </div>
+          <ul class="marginlistitems">
+            <li class="listitems">Javascript</li>
+            <li class="listitems">Ruby</li>
+            <li class="listitems">Html</li>
+            <li class="listitems">Css</li>
+          </ul>
+          <div class="thumbnail">
+            <div class="rectangle square">
+            </div>
+            <h1 class="headings">Frameworks</h1>
+          </div>
+          <ul class="marginlistitems">
+            <li class="listitems">Bootstrap</li>
+            <li class="listitems">Ruby on Rails</li>
+            <li class="listitems">RSPec</li>
+            <li class="listitems">CapyBara</li>
+            <li class="listitems">Selenium</li>
+          </ul>
+          <div class="thumbnail">
+            <div class="rectangle circle">
+            </div>
+            <h1 class="headings ">Skills</h1>
+          </div>
+          <ul class="marginlistitems">
+            <li class="listitems ">Codekit</li>
+            <li class="listitems">Github</li>
+            <li class="listitems">Codepen</li>
+            <li class="listitems">Gitlab</li>
+            <li class="listitems">Terminal</li>
+          </ul>
+        </section>
```

Comment:
- [ ] I kindly suggest that you also use a nested list for your technology, skills, and frameworks sections. The general container for the three of them should be a ul.

### Example
```
   <ul>
        <li class="languague">
              <ul>
                  <li><a href="#">JavaScript</a></li>
                  <li><a href="#">HTML</a>
                  <li><a href="#">CSS</a>
             </ul>
         </li>
          <li class="framework">
              <ul>
                  <li><a href="#">Bootstrap</a></li>
                  <li><a href="#">Ruby on Rails</a>
                  <li><a href="#">RSPEC</a>
             </ul>
         </li>
            <li class="skills">
              <ul>
                  <li><a href="#">Code Kit</a></li>
                  <li><a href="#">GitHub</a>
                  <li><a href="#">Codepen</a>
             </ul>
         </li>                                     
</ul>
```